src/components/Transcripts.tsx

```diff
@@ -98,6 +111,15 @@ function LoadedTranscripts({ transcripts: unsorted }: {
           </CardBody>
         </Card>
       }
+      {nameMapModal &&
+        <SummaryNameMapModal
+          nameMap={nameMap}
+          groupUsers={groupUsers}
+          onClose={() => {
+            handleNameMapModal();
+            refetchSummary();
+            refetchNameMap();
```

Comment:
no need for fetch calls. use trpcNext's invalidate method from within the `save` function in `SummaryNameMapModal`. Search the codebase for "invalidate" for references.